### DISCUSSION TITLE
Disable hold

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#2427] [Help with using Postal] [answered]
It seems that postal will move mails to hold, if the email address is on the bounce list. Is there a way to just make it fail instead? We send out time sensitive emails, so sending it again after 30 days is really a bad idea. Currently I need to manually remove things from hold, which I wish to avoid.

Edit: Just noticed that there is a `maximum_hold_expiry_days` setting in the config. I am not quite sure what the point is then. Why put on hold for 7 days if the suspension is longer than that?

**Comment by willpower232** [comment]
A message on hold can be easily released to its destination manually if you like, when the hold has expired then it would need resending from the original source. Postal will not send it otherwise.

The reason an email address is on the suppression list is because it did thoroughly fail at some point.

**Reply by sinnbeck** [reply]
But am I correct to assume that they are always put on hold due to the suppression list? And can I just make them fail instead?

**Reply by willpower232** [reply, marked as answer]
Unfortunately not, they're held because of the suppression list, they would only fail if Postal attempted to send the message (and they weren't successful).

**Reply by sinnbeck** [reply]
Ok so always due to suppression at least :) I have just set my webhook to mark Held as "failed due to suppression" and then set the expiry for hold to 1 day :)
